src/lib/containers/QuerySortSelector.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react'
+import { SortConfiguration } from './CardContainerLogic'
+import { useInfiniteQueryContext } from './QueryContext'
+import { SortDirection, SortItem } from '../utils/synapseTypes'
+import { unCamelCase } from '../utils/functions/unCamelCase'
+import Typography from '../utils/typography/Typography'
+import Creatable from 'react-select/creatable'
+import {
+  EnumOption,
+  findValueOption,
+  Control,
+} from './entity/annotations/CustomSelectWidget'
+
+export type QuerySortSelectorProps = {
+  sortConfig: SortConfiguration
+  facetAliases?: Record<string, string>
+}
+
+const QuerySortSelector: React.FunctionComponent<QuerySortSelectorProps> = ({
+  sortConfig,
+  facetAliases,
+}) => {
+  const { defaultColumn, defaultDirection, sortableColumns } = sortConfig
+  const infiniteQueryContext = useInfiniteQueryContext()
```

Comment:
Can just use `useQueryContext` here, because you aren't using any fields exclusive to the InfiniteQueryContext.

Reply:
Done